app/controllers/requests_controller.rb

```diff
@@ -0,0 +1,80 @@
+class RequestsController < ApplicationController
+  # TODO: Remove me, should be set somewhere else
+  before_action :set_current_user
+
+  def index
+    if @current_user.is_supervisor
+      @requests = Request.all
+    else
+      @requests = Request.where(user: User.where(is_supervisor: false))
```

Comment:
Is this the desired behavior? I can see anyone's requests for time off unless they're a supervisor? I'd be a little uncomfortable knowing my co-workers can see the times I'm requesting off. Unless @umayavuz says otherwise, I guess.